tools/flavor-convert/Makefile

```diff
@@ -0,0 +1,7 @@
+GITTAG := $(shell git describe --tags --abbrev=0 2> /dev/null)
+VERSION := $(or ${GITTAG}, v0.0.0)
+
+all:
+	rm -rf /opt/hvs-flavortemplates
```

Comment:
Similar to the flavor-gen, don't build things in the user's /opt directory.  Put the output of make in a subdir of the project.  On Windows...

$ make all
rm -rf /opt/hvs-flavortemplates
cp -r flavortemplates /opt/hvs-flavortemplates
**cp: cannot create directory '/opt/hvs-flavortemplates': No such file or directory**

Reply:
We have read the flavor template files from the flavortemplates folder that is the subdirectory of the project.